worlds/osrs/Options.py

```diff
@@ -58,6 +59,31 @@ class ProgressiveTasks(Toggle):
     display_name = "Progressive Tasks"
 
 
+class EnableDuds(Toggle):
+    """
+    Whether to include filler "Dud" checks that serve no purpose but allow for more tasks in the pool.
```

Comment:
A "check" is a location, not an item. I think these mean to say Dud items rather than checks.